scripts/mirror_apko.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+RAW=$(mktemp)
+
+REPOSITORY=${1:-"thesayyn/apko"}
```

Comment:
should replace your name here?

Reply:
filed https://github.com/chainguard-dev/rules_apko/issues/3